static/deploy/python/infer.py

```diff
@@ -557,15 +566,16 @@ def main():
         device=FLAGS.device,
         run_mode=FLAGS.run_mode,
         trt_calib_mode=FLAGS.trt_calib_mode,
-        enable_mkldnn=FLAGS.enable_mkldnn)
+        enable_mkldnn=FLAGS.enable_mkldnn,
+        enable_mkldnn_bfloat16=FLAGS.enable_mkldnn_bfloat16)
     if config.arch == 'SOLOv2':
         detector = DetectorSOLOv2(
             config,
             FLAGS.model_dir,
             device=FLAGS.device,
             run_mode=FLAGS.run_mode,
             trt_calib_mode=FLAGS.trt_calib_mode,
-            enable_mkldnn=FLAGS.enable_mkldnn)
```

Comment:
Here you removed enable_mkldnn, and I think both enable_mkldnn and enable_mkldnn_bfloat16 should remain

Reply:
Hi, you are right, I have changed it.